pkg/sqlstore/impl/system/internal/db/db.go

```diff
@@ -41,6 +41,12 @@ func Prepare(ctx context.Context, db DBTX) (*Queries, error) {
 	if q.getTablesByStructureStmt, err = db.PrepareContext(ctx, getTablesByStructure); err != nil {
 		return nil, fmt.Errorf("error preparing query GetTablesByStructure: %w", err)
 	}
+	if q.getIdStmt, err = db.PrepareContext(ctx, getId); err != nil {
```

Comment:
FYI, I've in my backlog to create a GH issue to move again to sqlc now that it supports SQLite. Doing this is annoying.